kill l3 agent

TregubovYury opened the pull request:
Author: Tregubov Yury

Added test QA-450. Also added primary_controller property in the
Environment class. Updated doc string for all my previous tests.

Commit:
kill l3 agent

Author: Tregubov Yury

Added test QA-450. Also added primary_controller property in the
Environment class. Updated doc string for all my previous tests.

Commit:
kill l3 agent

Author: Tregubov Yury

Added test QA-450. Also added primary_controller property in the
Environment class. Updated doc string for all my previous tests.

AlexandraAllakhverdieva committed:
add test_543176_CreatingMultipleSnapshots

AlexandraAllakhverdieva committed:
Merge remote-tracking branch 'upstream/master'

AlexandraAllakhverdieva committed:
fix pep8

tkuterina committed:
added test_2238776_NetworkConnectivityToVMDuringLiveMigration

tkuterina committed:
Merge remote-tracking branch 'upstream/master'

tkuterina committed:
fix issue after check

AlexandraAllakhverdieva committed:
Merge remote-tracking branch 'upstream/master'

tkuterina committed:
test in progress

tkuterina committed:
merge update

tkuterina committed:
fix issue

Commit:
kill dhcp agent

Author: Tregubov Yury

Added test for the QA-458
Removed non ascii symbols in doc comment for QA-451

Commit:
kill dhcp agent

Author: Tregubov Yury

Added test for the QA-458
Removed non ascii symbols in doc comment for QA-451

TimurNurlygayanov commented on mos_tests/neutron/python_tests/test_restarts.py:
```diff
@@ -71,21 +74,37 @@ def prepare_openstack(self, setup):
+                                    binary='neutron-dhcp-agent')['agents']]
+
     def test_shutdown_primary_controller_with_l3_agt(self):
+        """Id of the test is C57896
```
"""Id of the test is C57896      - Let's add human readable description of this test case.

TimurNurlygayanov commented on mos_tests/neutron/python_tests/test_restarts.py:
```diff
@@ -71,21 +74,37 @@ def prepare_openstack(self, setup):
+
     def test_shutdown_primary_controller_with_l3_agt(self):
+        """Id of the test is C57896
+            8. Check on what agents is router1:
```
Let's start from #1

TregubovYury replied:
The numeration is stared in the fixture in the same class. Since the first actions are common for all tests.

TimurNurlygayanov commented on mos_tests/neutron/python_tests/test_restarts.py:
```diff
@@ -122,6 +141,22 @@ def test_shutdown_primary_controller_with_l3_agt(self):
 
     def test_restart_primary_controller_with_l3_agt(self):
+        """Id of the test is C57893
+            8. Check on what agents is router1:
```
the same

TimurNurlygayanov commented on mos_tests/neutron/python_tests/test_restarts.py:
```diff
@@ -141,8 +176,57 @@ def test_restart_primary_controller_with_l3_agt(self):
+
+    def test_kill_active_l3_agt(self):
+        """Id of the test is C57892
+            8. get node with l3 agent where is the router1:
```
the same

TimurNurlygayanov commented on mos_tests/neutron/python_tests/test_restarts.py:
```diff
@@ -150,3 +234,87 @@ def test_restart_primary_controller_with_l3_agt(self):
+
+    def test_kill_active_dhcp_agt(self):
+        """Id of the test is C57907
+            8. Log on instacne by root:
```
the same

TimurNurlygayanov commented on mos_tests/neutron/python_tests/test_dhcp_agent.py:
```diff
@@ -62,6 +56,23 @@ def isclose(self, a, b, rel_tol=1e-9, abs_tol=0.0):
         return abs(a - b) <= max(rel_tol * max(abs(a), abs(b)), abs_tol)
 
     def test_to_check_dhcp_agents_work(self):
+        """Id of the test is C57900
```
Let's add human readable description of test case and let's start steps from 1.

AlexandraAllakhverdieva committed:
Fix remarks

AlexandraAllakhverdieva committed:
Merge

gdyuldin commented on mos_tests/neutron/python_tests/test_restarts.py:
```diff
@@ -71,21 +74,37 @@ def prepare_openstack(self, setup):
-                self.primary_host = node.data['fqdn']
-                break
+        self.primary_node = DevopsClient.get_devops_node(
+            self.env.primary_controller.data['name'].split('_')[0])
```
If you deploy OpenStack by hand - name of nodes inside fuel may be any string. I believe it will better to determine devops node by MAC address.

TimurNurlygayanov replied:
+1

tkuterina committed:
fix issue

Commit:
Merge pull request #61 from gdyuldin/pytest

Switch to pytest mark to check test requirements

Commit:
Merge pull request #62 from gdyuldin/QA-337

Add testcase 'Check the tunnels established l2pop'

Commit:
Merge pull request #53 from AlexandraAllakhverdieva/master

add test_543176_CreatingMultipleSnapshots

tkuterina committed:
merge

tkuterina committed:
merge

tkuterina committed:
merge

Commit:
Merge pull request #54 from tkuterina/master

test_2238776_NetworkConnectivityToVMDuringLiveMigration

Commit:
Merge pull request #63 from gdyuldin/QA-337

Fix logger argument

Commit:
Merge pull request #64 from gdyuldin/QA-363

Add testcase 'Broadcast traffic between instances'

Commit:
drop rabbit port check dhcp agent

Author: Tregubov Yury

Added test for the QA-456
Applied comments for previous commits and did some refactoring

Commit:
drop rabbit port check dhcp agent

Author: Tregubov Yury

Added test for the QA-456
Applied comments for previous commits and did some refactoring

Commit:
Merge branch 'master' of https://github.com/TregubovYury/mos-integration-tests

Commit:
fixed merge confilict

Author: Tregubov Yury

Fixed merge conflict that left in  os_actions

rpromyshlennikov commented on mos_tests/neutron/python_tests/test_dhcp_agent.py:
```diff
@@ -12,9 +12,10 @@
 import pytest
+import time
+
+import logging
```
import order mismatch: 1st built-ins, like logging, than \n, than requirements(side packages), than \n, than self package

rpromyshlennikov commented on mos_tests/neutron/python_tests/test_restarts.py:
```diff
@@ -12,14 +12,13 @@
-
 import pytest
 
+import logging
```
same as upper comment

rpromyshlennikov commented on mos_tests/nova/nova_tests.py:
```diff
@@ -14,6 +14,7 @@
 import os
 import unittest
+import subprocess
 from time import time, sleep
```
import order o < s < t < u

rpromyshlennikov commented on mos_tests/neutron/python_tests/test_dhcp_agent.py:
```diff
@@ -55,15 +50,40 @@ def prepare_openstack(self, setup):
+                                    binary='neutron-dhcp-agent')['agents']]
+
+    def run_udhcpc_on_vm(self, vm):
+        command = 'sudo -i udhcpc'
```
This command is deprecated on cirros:
use sudo -i cirros-dhcpc up eth0 instead

Commit:
applied comments

Author: Tregubov Yury

Applied comments from Rodion

rpromyshlennikov commented on mos_tests/cinder/cinder_tests.py:
```diff
@@ -0,0 +1,139 @@
+
+from keystoneclient.v2_0 import client as keystone_client
+from novaclient import client as nova_client
+from cinderclient import client as cinder_client
```
c < k < n
